app/routes/docs.contributing.github.mdx

```diff
@@ -0,0 +1,53 @@
+# GitHub
+
+For all projects in the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization, contributors should adhere to the following steps:
+
+- Fork a copy of our repo
+
+- Clone your repo to your machine
+
+- Create a branch for your new changes
+
+- Create a Pull Request to contribute to the original project
+
+For more information, check out our [web-dev-bootcamp](https://github.com/nasa-gcn/web-dev-bootcamp) repository. In the syllabus folder you will find some instructive lectures and examples to help understand our workflow.
+
+## Forking a Repo
+
+Navigate to the page repository page for whichever project you wish to contribute to, for example: [gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov)
+
+Click the Fork button
+
+Click Create Fork to create a new repo in your account
+
+## Cloning a Repo
+
+We recommend checking out [this guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on adding an SSH key to your github account.
+
+With your new fork now created, click the Code button and copy the URL in the SSH tab.
+
+Open a terminal window and run the following
+
+```sh
+git clone <your copied repo url>
+```
+
+## Branching off main
+
+Within your cloned repo on your local machine, run the following to create and switch to a new branch for your code changes:
```

Comment:
"Within" is a little vague. I understand that you mean "in a terminal with the current working directory set to the clone directory," but that's not clear from the text.